server/app.js

```diff
@@ -2,8 +2,8 @@ const express = require("express");
 const path = require("path");
```

Comment:
Path isn't necessary anymore now that we're no longer serving our front-end within our express server as well!